src/presentation/pages/NoteSettings.vue

```diff
@@ -210,4 +250,16 @@ onMounted(async () => {
   gap: var(--spacing-xxl);
   margin: var(--spacing-xxl) 0;
 }
+
+.change-parent{
+  display: flex;
+  flex-direction: column;
+  gap: var(--v-padding);
+
+  &__button{
+    display: flex;
+    justify-content: flex-end;
+    flex-grow: 1;
+  }
```

Comment:
i think card slot alignment was fixed in some of previous pr's, try to update from main and remove this style

Reply:
Yes, it was fixed. This style was removed in last commit